### PR TITLE
modified the card using custom css

### PR DESCRIPTION
**Pull request by karthikgeek1**

### CODE REVIEW

File: src/Components/Cards/Cards.jsx [hunk 1]
```diff
@@ -1,34 +1,20 @@
...
-                style={{ backgroundColor: '#111827', width: "280px", height: "200px", borderRadius: '10px' }}
-                extra={<img src='https://pbs.twimg.com/profile_images/1145524454170062848/U4lxVYEw_400x400.png'
-                    style={{ height: "50px", width: "50px", borderRadius: '50%' }} />}
-            >
-                <h3 >price:<span> $64.32</span></h3>
-            </Card>
-        {/* </div> */}
+            <div className='card'>
+                <div className='stockdata'>
+                    <img src='https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcT8q0metut-4bQutjD3LQhql0bfSaXE_4ETJQ&usqp=CAU' style={{ height: "30px", width: "30px", borderRadius: '50%' }} />
```
Comment (Souvikdeb2612) [inline src/Components/Cards/Cards.jsx]: remove inline styling

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/Cards/Cards.jsx (hunk 1 above)
Comment (karthikgeek1) [reply src/Components/Cards/Cards.jsx]: solved

File: src/Components/Cards/Cards.jsx [hunk 2]
```diff
@@ -1,34 +1,20 @@
-import { Card, Collapse } from 'antd';
 import React from 'react';
 import './Cards.css'
```
Comment (Souvikdeb2612) [inline src/Components/Cards/Cards.jsx:3]: 1 line space before css imports

**Review by Souvikdeb2612 — CHANGES_REQUESTED** [review]

File: src/Components/Cards/Cards.jsx (hunk 2 above)
Comment (karthikgeek1) [reply src/Components/Cards/Cards.jsx:3]: solved

**Review by Souvikdeb2612 — APPROVED** [review]

**Pull request merged**